docs/middleware/op-submission.md

```diff
@@ -157,3 +157,24 @@ backend.use('apply', (request, next) => {
 
 {: .warn :}
 The `request.$fixup()` method may throw an error, which should be handled appropriately, usually by passing directly to the `next()` callback.
+
+## Comparing old snapshot version with new version
+
+Frequently, it becomes necessary to verify the changes made. This can be accomplished by leveraging two hooks, `apply` and `commit`, and creating a snapshot clone within the `apply` hook.
+
+```js
+backend.use('apply', (request, next) => {
+  // Here use your favorite deep clone implementation
+  // sharedb team recommends rfdc
+  // https://github.com/davidmarkclements/rfdc#readme
```

Comment:
I think let's skip the recommendation here; `sharedb` doesn't actually use this internally anywhere.